### PR TITLE
Answer:8 

### PR DESCRIPTION
**Pull request by balaji-balamurugan**
## Checklist for challenge submission

- [ ] Start your PR message with Answer:${challenge_number}

### CODE REVIEW

File: apps/pipe-easy/src/app/app.component.ts [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+import { Component, Pipe, PipeTransform } from '@angular/core';
+@Pipe({
+  name: 'compute',
+  pure: true,
   standalone: true,
-  imports: [NgFor],
+})
+export class HeavyComputationPipe implements PipeTransform {
+  transform(value: { name: string; index: number }) {
+    return `${value?.name} - ${value?.index}`;
```
Comment (tomalaforge) [inline apps/pipe-easy/src/app/app.component.ts:10]: never really though of passing my value this way, but this is working. Doing this way, if name of index change, both will be recompute. In our case this is what we want, so this is not an issue. But if you have static values you should consider doing like the following: 

the most common way is to have multiple property like 
```typescript
transform(name: string, index: number ) {
```

and in your html, you call your pipe like this:
```html
arg1 | mypipe : arg2
```

**Comment by tomalaforge** [comment]
thanks for submitting your PR. nice work 👍

**Comment by balaji-balamurugan** [comment]
thanks for marking it as the answer @tomalaforge

File: apps/pipe-easy/src/app/app.component.ts (hunk 1 above)
Comment (balaji-balamurugan) [reply apps/pipe-easy/src/app/app.component.ts:10]: yeah okay understood @tomalaforge

**Pull request closed without merging**